### PR TITLE
Fix / Start date format cart toast

### PR DESCRIPTION
**Pull request by amandaflod**
## Describe your changes

Update the start date description in the cart toast to include cancellation. 

Screenshot 2023-02-10 at 15 39 38

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/components/CartNotification/ProductItem.tsx [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
-import { useTranslation } from 'react-i18next'
 import { Space, Text } from 'ui'
 import { Pillow } from '@/components/Pillow/Pillow'
 import { SpaceFlex } from '@/components/SpaceFlex/SpaceFlex'
 
 export type ProductItemProps = {
   name: string
   price: string
-  startDate?: string
+  startDateDescription?: string
```
Comment (robinandeer) [inline apps/store/src/components/CartNotification/ProductItem.tsx:9]: I would consider calling it just "description" to make it more flexible

File: apps/store/src/components/ProductPage/PurchaseForm/PurchaseForm.tsx [hunk 2]
```diff
@@ -102,8 +102,12 @@ export const PurchaseForm = () => {
             notifyProductAdded({
               name: productData.displayNameFull,
               price: formatter.monthlyPrice(item.price),
-              /* @TODO: display "automatically switches" if cancellation is requested" */
-              startDate: formatter.fromNow(new Date(item.startDate)),
+              startDateDescription: item.cancellation.requested
+                ? t('CART_ENTRY_AUTO_SWITCH', { ns: 'cart' })
```
Comment (robinandeer) [inline apps/store/src/components/ProductPage/PurchaseForm/PurchaseForm.tsx:106]: nit: consider moving this translation key from "cart" to "purchase-form" namespace

**Review by robinandeer — APPROVED** [review]
Very nice! Only thing I would consider changing is updating the new prop name

**Review by guilhermespopolin — APPROVED** [review]
Well done :)

File: apps/store/src/components/CartNotification/ProductItem.tsx (hunk 1 above)
Comment (amandaflod) [reply apps/store/src/components/CartNotification/ProductItem.tsx:9]: okidoki, will do!

File: apps/store/src/components/ProductPage/PurchaseForm/PurchaseForm.tsx (hunk 2 above)
Comment (amandaflod) [reply apps/store/src/components/ProductPage/PurchaseForm/PurchaseForm.tsx:106]: The thing is that we're using it in the CartEntryItem as well, which definitely belongs to cart 🤔 Should I create a duplicate maybe?

**Pull request merged**